static/schemas/2.0/user.2.0.json

```diff
@@ -0,0 +1,30 @@
+{
+    "title": "User.2.0",
+    "definitions": {
+        "PartialUser": {
+            "description": "A RackHD local user",
+            "type": "object",
+            "properties": {
+                "username": {
+                    "description": "The username of the user",
+                    "type": "string"
+                },
+                "password": {
+                    "description": "The password of the user",
+                    "type": "string"
+                },
+                "role": {
```

Comment:
The `role` is to specify the read/write right? If so, change the role to `enum` will be better.